pyproject.toml

```diff
@@ -5,14 +5,23 @@ description = "Cookiecutter template for baking Nautobot App cookies."
 authors = ["Network to Code, LLC <info@networktocode.com>"]
 readme = "README.md"
 repository = "https://github.com/nautobot/cookiecutter-nautobot-app"
+classifiers = [
+    "Intended Audience :: Developers",
+    "Development Status :: 5 - Production/Stable",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+]
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.dependencies]
 cookiecutter = "^2.3.0"
-python = "^3.8"
+python = ">=3.9,<3.13"
```

Comment:
This seems overly strict for the cookie? Python 3.13 is being shipped by default in all of the major linux distributions.

Reply:
This is for the cookie env, I would argue that we can in fact only support the latest in the development environment.

Can you clarify the concern?

Reply:
To be clear, I agree that we should change the supported Python versions for the baked app's pyproject.toml but this is the pyproject.toml for the cookiecutter.

Reply:
> This means that most users would have to install another version of python just so that they can bake the cookie.

This is a very small community, I am not sure I understand the concern.